yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -130,11 +131,11 @@ func prepareCopyCommands() {
 				if err != nil {
 					utils.ErrExit("opening datafile %q to prepare copy command: %v", filePath, err)
 				}
-				copyTableFromCommands[table] = fmt.Sprintf(`COPY %s(%s) FROM STDIN WITH (FORMAT %s, DELIMITER '%c', ESCAPE '%s', QUOTE '%s', HEADER,`,
+				copyTableFromCommands[table] = fmt.Sprintf(`COPY %s(%s) FROM STDIN WITH (FORMAT %s, DELIMITER '%c', ESCAPE E'%s', QUOTE E'%s', HEADER,`,
```

Comment:
do we need to use `E` for `delimiter` also here

Reply:
No, that is fine I tried that without E as well these characters are working fine. It is not required for such characters.

Reply:
As discussed, added it for being on the safer side and uniformity.